README.md

```diff
@@ -526,7 +526,7 @@ These colorschemes may not specialize in Tree-sitter directly but are written in
 - [alvarosevilla95/luatab.nvim](https://github.com/alvarosevilla95/luatab.nvim) - A simple tabline written in Lua.
 - [johann2357/nvim-smartbufs](https://github.com/johann2357/nvim-smartbufs) - Smart buffer management.
 - [kdheepak/tabline.nvim](https://github.com/kdheepak/tabline.nvim) - A "buffer and tab" tabline.
-- [noib3/cokeline.nvim](https://github.com/noib3/cokeline.nvim) - A Neovim bufferline for people with addictive personalities.
+- [willothy/nvim-cokeline](https://github.com/willothy/nvim-cokeline) - A Neovim bufferline for people with addictive personalities.
```

Comment:
```suggestion
- [willothy/nvim-cokeline](https://github.com/willothy/nvim-cokeline) - A bufferline for people with addictive personalities.
```

Thanks for the PR! I know you didn't change this line but I would like to take the opportunity to improve it.

Reply:
Thanks!